src/Elasticsearch/Product/SearchFieldConfig.php

```diff
@@ -53,12 +53,17 @@ public function usePrefixMatch(): bool
 
     public function getFuzziness(string $token): string|int
     {
-        $fuzziness = $this->tokenize ? 'auto' : 1;
-
+        // Block fuzziness for numeric/serial-like
```

Comment:
```suggestion
        // Disable fuzziness for numeric tokens or a serial of at least 3 digits
```

can't we re-use the comment from within the if condition? it is still valid, right? 
and the thing with at least 3 digits sounds important to know